src/build/create-buildinfo.groovy

```diff
@@ -0,0 +1,56 @@
+/*
```

Comment:
What do you think about using the git commit I'd plugin instead of this groovy and we can also include the git hash in the user agent (can be useful for snapshot users)?

Reading the properties will be slightly more complicated than just referencing this class though.

https://github.com/ktoso/maven-git-commit-id-plugin